prime-router/src/main/resources/db/migration/V32__action_time_and_sender_indexes.sql

```diff
@@ -0,0 +1,23 @@
+/*
+ * The Flyway tool applies this migration to create the database.
+ *
+ * Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+ * use VARCHAR(63) for names in organization and schema
+ *
+ * Copy a version of this comment into the next migration
+ *
+ */
+
+-- These indexes already exist on both staging and production, including
```

Comment:
I do not think you need this comment here and on the next file since it will only be valid once ... now.

Reply:
Good point! Will remove.